src/document/DocumentAnnotator.ts

```diff
@@ -97,10 +97,17 @@ export default class DocumentAnnotator extends BaseAnnotator {
             return;
         }
 
-        if (mode === Mode.HIGHLIGHT) {
-            this.annotatedEl.classList.add('ba-is-highlighting');
-        } else {
-            this.annotatedEl.classList.remove('ba-is-highlighting');
+        switch (mode) {
```

Comment:
Maybe we could remove all `ba-is-*` classes regardless of mode? Something like `['ba-is-highlighting', 'ba-is-create-region'].forEach(className => this.annotatedEl.classList.remove(className))`

Reply:
I think this can work.